src/check-header-guards.sh

```diff
@@ -11,6 +11,7 @@ test "x$HBSOURCES" = x && HBSOURCES=`cd "$srcdir"; find . -maxdepth 1 -name 'hb-
 
 for x in $HBHEADERS $HBSOURCES; do
 	test -f "$srcdir/$x" -a ! -f "$x" && x="$srcdir/$x"
+	test "$x" = "./hb-gobject-structs.h" && continue;
```

Comment:
@behdad, hadn't chance to fully digest the check, can you see what has went wrong `./hb-gobject-structs.h`?

Reply:
Ok, couldn't digest it at 1AM but now can :) This seem OK, clever use of compiler directive and definition for preventing duplication on headers btw